### PR TITLE
Allow geocoding result selection in EventCreator

### PR DESCRIPTION
**Pull request by Ef55**

### CODE REVIEW

**Comment by Ef55** [comment]
<details> 
<summary>Before searching: </summary>

</details>

<details> 
<summary>After searching: </summary>

</details>

<details> 
<summary>After result selection: </summary>

</details>

**Comment by Ef55** [comment]
The location search is now done "in real time" (no need to click the search button). When enabled, the phone location is just added as an additional result (at the end).

**Screenshots**:
<details>
<summary>Before searching: </summary>
</img>
</details>

<details>
<summary>After seach: </summary>
</img>
</details>

<details>
<summary>With phone location enabled: </summary>
</img>
</details>

File: app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java [hunk 1]
```diff
@@ -56,14 +75,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
+        ////////////////////////////////
+        //  Attributs initialization  //
+        ////////////////////////////////
```
Comment (justagoodguy-omega) [inline app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java:80]: image

File: app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java [hunk 2]
```diff
@@ -73,6 +97,15 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
...
         this.validate = getView().findViewById(R.id.buttonSetGeolocation);
 
+        // Setup recycler view
+        this.locationList = (RecyclerView) getView().findViewById(R.id.locationMatchesList);
+        this.locationList.setLayoutManager(new LinearLayoutManager(getActivity()));
+
+
+        ////////////////////////////////
+        //         Logic setup        //
+        ////////////////////////////////
```
Comment (justagoodguy-omega) [inline app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java:107]: image

**Review by justagoodguy-omega — APPROVED** [review]
Nice

File: app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java (hunk 1 above)
Comment (hugolan) [reply app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java:80]: ???

File: app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java (hunk 2 above)
Comment (hugolan) [reply app/src/main/java/com/github/onedirection/events/EventCreatorGeolocationFragment.java:107]: interesting review

**Review by hugolan — APPROVED** [review]

**Pull request merged**